composer.json

```
@@ -33,14 +43,14 @@
       }
     ],
     "require": {
-        "php": ">=7.1.0",
-        "sofa/eloquence-base": "dev-7.1.2-erasys as 7.1.2"
+        "php": ">=8.2",
+        "sofa/eloquence-base": "8.0.0"
```

I think the version should be the same

```suggestion
        "sofa/eloquence-base": "8.0.1"
```